pages/common/rabin2.md

```diff
@@ -0,0 +1,24 @@
+# rabin2
+
+> Get information about binary files - symbols, sections, linked libraries, etc.
+> Comes bundled with radare2.
+
+- Display general information about a binary (architecture, type, endianness):
+
+`rabin2 -I {{path/to/binary}}`
+
+- Display linked libraries:
+
+`rabin2 -l {{path/to/binary}}`
+
+- Display symbols imported from libraries:
+
+`rabin2 -i {{path/to/binary}}`
+
+- Display strings contained in the binary:
+
+`rabin2 -z {{path/to/binary}}`
+
+- Output in JSON (works for every subcommand):
```

Comment:
Maybe it's better to say `Display the output in JSON:` and remove the -I option. You can combine any of the flags with other commands also (Please correct me if I am wrong). So I guess it is a bit redundant to say that.

Reply:
You are right, but the `-j` option does not work on its own (it requires another flag), and I don't want to include examples that are broken.

Reply:
Oh I see. Alright then, just change the description. No need to remove `-I`

Reply:
I have changed the description in the latest commit.

Reply:
I also wanted to remove the `(works for every subcommand)`. Because you can intermix the previous flags also.

Reply:
Done.

Reply:
Thanks !